go.mod

```diff
@@ -3,10 +3,11 @@ module github.com/Layr-Labs/eigenda-proxy
 go 1.21
 
 require (
-	github.com/Layr-Labs/eigenda v0.8.1
+	github.com/Layr-Labs/eigenda v0.8.4-0.20240918205210-351526af1e80
```

Comment:
we should update to a proper release tag when doing the proxy release

Reply:
so this implies, eigenda proxy release has to delay before eigenda-core release

Reply:
gotcha. you'll keep track of this I'm guessing?